bot/exts/fun/duck_pond.py

```diff
@@ -145,6 +145,10 @@ async def on_raw_reaction_add(self, payload: RawReactionActionEvent) -> None:
         amount of ducks specified in the config under duck_pond/threshold, it will
         send the message off to the duck pond.
         """
+        # Ignore DMs.
+        if payload.guild_id is None:
```

Comment:
I wanted to say that it may make more sense to check whether the ID matches the configured PyDis ID, but in practice it probably makes no difference.